src/main/scala/com/typesafe/sbt/osgi/OsgiKeys.scala

```diff
@@ -103,6 +103,10 @@ object OsgiKeys {
     SettingKey[Boolean](prefix("FailOnUndecidedPackage"), "Fail the build if a package is neither exported or private." +
       "Without this setting such classes might be just transparently removed from the resulting artifact!")
 
+  val packageWithJVMJar: SettingKey[Boolean] =
+    SettingKey[Boolean](prefix("PackageWithJVMJar"), "Use the JVM jar tools to craft the bundle instead of the one from BND." +
+      "Without this setting the produced bundle are detected as corrupted by recent JVMs")
+
```

Comment:
This is the jar tool that comes with JVM, and we cant use BND because it creates corrupted jar's (thats why this feature was implemented in the first place)